ui/src/importer/importerTaskDetails/ImportedTable.js

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Pagination, Table } from 'semantic-ui-react';
+import _isEmpty from 'lodash/isEmpty';
+import { EitemImportDetailsModal } from '../EitemImportDetailsModal';
+import { SeriesImportDetailsModal } from '../SeriesImportDetailsModal';
+import { JsonViewModal } from '../JsonViewModal';
+import { ImportedDocumentReport } from './ImportedDocumentReport';
+
+export class ImportedTable extends React.Component {
+  constructor(props) {
+    super(props);
+    this.pageSize = 100;
+    const { records } = this.props;
```

Comment:
I really don't like storing the previous records to see if I should reset to page 1, but I run out of ideas so maybe I can rethink it with a bit of feedback. I don't know either if there is a way to destructure the records inside an object in which I have other fixed values, but I wasn't able.